### PR TITLE
[EventCollection] fix default value (0) latitudes and longitudes

### PR DESCRIPTION
**Pull request by chejennifer**
- when `optional` is not set for a proto field, default values (in this case, 0) will be dropped

https://protobuf.dev/programming-guides/field_presence/#presence-in-proto3-apis
TODO: we probably want to set other numeric values in the protos also to `optional`

### CODE REVIEW

**Comment by shifucun** [comment]
Just to confirm, this means all the lat,long would be set in the cache. Otherwise it can be incorrectly represented as 0.

**Review by shifucun — APPROVED** [review]

File: proto/v1/event.proto [hunk 1]
```diff
@@ -20,8 +20,8 @@ package datacommons.v1;
 message EventCollection {
   message GeoLocation {
     message Point {
-      double latitude = 1;
-      double longitude = 2;
+      optional double latitude = 1;
```
Comment (beets) [inline proto/v1/event.proto:23]: please add a note here so we don't drop it in the future

**Review by beets — COMMENTED** [review]
thx for the fix!

File: proto/v1/event.proto (hunk 1 above)
Comment (chejennifer) [reply proto/v1/event.proto:23]: done in a followup PR: https://github.com/datacommonsorg/mixer/pull/1025

**Pull request merged**